WRQ-31515: Fixed Scroller and VirtualList to set prop value to default value when `undefined` is passed for the prop value (#808)
enactjs/agate: CHANGELOG.md

### Fixed

> - `agate/Scroller`, `agate/VirtualList` and `agate/VirtualList.VirtualGridList` to set prop value to default when `undefined` is passed for the prop value

Reviewer: ```suggestion
- `agate/Scroller` and `agate/VirtualList` to have default prop  when `undefined` prop is passed
```

Author: It's fixed.
